src/main/resources/xslt/mods2schemaorg.xsl

```diff
@@ -1,836 +1,892 @@
 <?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
I am not really qualified for reviewing this. I just gave it a cursory glance and hope that the nice unit test covers it well enough.